### ISSUE TITLE
The submenu is shown wrongly on the first time

### ISSUE THREAD

**Issue by trmdi** [fcitx/fcitx-unikey#27]
Please see the screenshot:
![image](https://user-images.githubusercontent.com/11857205/141038295-036d8933-29d3-4900-a4c0-92b09b818944.png)

If you hide then show it again, it appears properly.
![image](https://user-images.githubusercontent.com/11857205/141038434-a8642848-47ca-4b3f-bfad-57718707e3e8.png)

I'm using the latest Qt on openSUSE Tumbleweed.

**Comment by wengxt** [comment]
To be honest, fcitx4 is not developed actively (I may sometimes fix random some bug).

Since you're already using tumbleweed, I suggest you upgrade to fcitx5 instead. If you have the same problem, please report to fcitx/fcitx5-unikey

**Comment by trmdi** [comment]
> To be honest, fcitx4 is not developed actively (I may sometimes fix random some bug).
> 
> Since you're already using tumbleweed, I suggest you upgrade to fcitx5 instead. If you have the same problem, please report to fcitx/fcitx5-unikey

Yes, it still happens with fcitx5-unikey: https://github.com/fcitx/fcitx5-unikey/issues/8
